text/accumulation.tex

```diff
@@ -141,7 +141,7 @@ \subsection{Execution}
       \mathbf{s} &= \{ \mathbf{r}_s \mid w \in \mathbf{w}, \mathbf{r} \in w_\mathbf{r} \} \cup \keys{\mathbf{f}} \\
       \mathbf{u} &= [(s, \Delta_1(\mathbf{o}, \mathbf{w}, \mathbf{f}, s)_u) \mid s \orderedin \mathbf{s}] \\
       \mathbf{b} &= \{ (s, b) \mid s \in \mathbf{s},\, b = \Delta_1(\mathbf{o}, \mathbf{w}, \mathbf{f}, s)_b,\, b \ne \none \} \\
-      \mathbf{t} &= [\Delta_1(\mathbf{o}, \mathbf{w}, \mathbf{f}, s)_\mathbf{t} \mid s \orderedin \mathbf{s}] \\
+      \wideparen{\mathbf{t}} &= [\Delta_1(\mathbf{o}, \mathbf{w}, \mathbf{f}, s)_\mathbf{t} \mid s \orderedin \mathbf{s}] \\
```

Comment:
Incorrect. Deferred transfers are $\wideparen{\mathbf{t}}$ (a sequence of transfers), but this term is just $\mathbf{t}$ (a sequence of sequences of transfers).

Reply:
reverted in [b60777](https://github.com/gavofyork/graypaper/pull/323/commits/1b60777eb13d6f878c714ccf623e6a0c01968c5d)